feature/experimental/qos/ingress_traffic_classification_and_rewrite/README.md

```diff
@@ -0,0 +1,83 @@
+# QoS-1.1: Ingress traffic classification and rewrite
+
+## Summary
+
+This test aims to validate the functionality of ingress traffic classification and subsequent packet remarking (rewrite) on a Device Under Test (DUT). The DUT's configuration will be evaluated against the OpenConfig QOS model, and traffic flows will be analyzed to ensure proper classification, marking, and forwarding.
+
+## Procedure
+
+* DUT Configuration:
+    * Configure the DUT's ingress and egress interfaces.
+    * Apply QoS classifiers using the OpenConfig QOS model, matching packets based on DSCP/TC/EXP values as per the classification table.
+    * Configure packet remarking rules based on the marking table.
```

Comment:
* Set up required static MPLS LSP for testing MPLS swap and pop actions.
* Set up required GRE decapsulation policy-forwarding rules required for GRE decap classification testing.